### DISCUSSION TITLE
On Project / View added columns disappearing after navigating away and coming back

### DISCUSSION THREAD

**Discussion by kozaka-tv** [community/community#33482] [Projects and Issues] [answered]
I added to a view a column: milestone

![image](https://user-images.githubusercontent.com/4964462/190990668-6d9149b1-449e-470b-84cb-040c0a99646e.png)

Then I went back to projects, and selected the same project. The column is gone:

![image](https://user-images.githubusercontent.com/4964462/190990871-c855f36b-3ff0-479c-87bb-18ee72de917c.png)

So I have to set it always if I enter to the view. It would be great, that this stays permanent.

**Comment by rileybroughten** [comment, marked as answer]
Hello @kozaka-tv ! After you add a new column you can [save the view](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/managing-your-views#saving-changes-to-a-view) so that these changes persist!

**Comment by kozaka-tv** [comment]
Oh, thank you! I do not know that this is new feature, but I found it a week ago. So everything is fine now!
